### ISSUE TITLE
Registry down

### ISSUE THREAD

**Issue by CH4RON** [xoofx/UnityNuGet#251]
https://unitynuget-registry.azurewebsites.net/registry.json

returns
{"error":"not_initialized","reason":"The server is initializing (0.3% completed). Please retry later..."}

**Comment by bdovaz** [comment]
It's initializing...

https://unitynuget-registry.azurewebsites.net/status

**Comment by bdovaz** [comment]
@xoofx the issue of it going very slow is because the machine in Azure is the free tier for being MVP, no? Because GitHub workflows do a conversion of all packages and it's in 1h 10-20min right now.

The full bootstrap (caused by the registry version change) in Azure takes hours.

**Comment by xoofx** [comment]
> the machine in Azure is the free tier for being MVP

I haven't checked, I thought that initially I took a different machine... but recently they messed up with my Visual Studio MVP Azure subscription, and I have been basically charged for €140/month for the past 6 months... on top of that, my credit card expired, and I'm dealing with Azure support for get refunded. 😒

**Comment by bdovaz** [comment]
The registry is up again @CH4RON

**Comment by CH4RON** [comment]
Thanks!

**Comment by GAnfs** [comment]
It seems https://unitynuget-registry.azurewebsites.net/status is down again (403)

**Comment by xoofx** [comment]
Yes, it has reached the expense limits for this month. It should reset tonight or tomorrow.

**Comment by p98m** [comment]
@xoofx What is the current size of `unity_packages` folder at your server? Could you provide archive of this folder to public with s3/torrent/anything?
